turbopack/crates/turbo-persistence/src/write_batch.rs

```diff
@@ -36,14 +36,14 @@ struct ThreadLocalState<K: StoreKey + Send, const FAMILIES: usize> {
     /// The collectors for each family.
     collectors: [Option<Collector<K, THREAD_LOCAL_SIZE_SHIFT>>; FAMILIES],
     /// The list of new blob files that have been created.
-    new_blob_files: Vec<File>,
+    new_blob_files: Vec<(u32, File)>,
```

Comment:
This should either use a named struct, or it should include a comment about what the `u32` is used for.